sdk/core/azure-core-tracing-opencensus/azure/core/tracing/ext/opencensus_span/__init__.py

```diff
@@ -82,7 +82,7 @@ def __init__(self, span=None, name="span", **kwargs):
             raise ValueError("Kind {} is not supported in OpenCensus".format(value))
 
         links = kwargs.pop("links", None)
-        self._span_instance = span or tracer.start_span(name=name, **kwargs)
+        self._span_instance = span or tracer.start_span(name=name)
```

Comment:
Do we really want to ignore all kwargs in order to avoid unexpected keyword argument?

Reply:
In this case, the `start_span` method for opencensus never actually accepted any kwargs besides `name` which we explicitly pass in. Can see here: https://github.com/census-instrumentation/opencensus-python/blob/master/opencensus/trace/tracer.py#L106

So the only way this method wouldn't break is if `kwargs` was empty.